scripts/isct_convert_binary_to_trilinear.py

```diff
@@ -131,7 +147,7 @@ def main():
     # Delete temporary files
     if remove_temp_files == 1:
         print '\nRemove temporary files...'
-        sct.run('rm -rf '+ path_tmp)
+        shutil.rmtree(path_tmp)
```

Comment:
I would put `shutil.rmtree(path_tmp,True)` to let go error. Then we get the a behaviour closer to `rm -rf`.